src/openms/source/CHEMISTRY/IsotopeDistribution.cpp

```diff
@@ -199,6 +199,61 @@ namespace OpenMS
     }
   }
 
+
+  void IsotopeDistribution::estimateFromRNAWeight(double average_weight)
+  {
+      estimateFromWeightAndComp(average_weight, 9.75, 12.25, 3.75, 7, 0, 1);
+  }
+
+
+  void IsotopeDistribution::estimateFromDNAWeight(double average_weight)
+  {
+      estimateFromWeightAndComp(average_weight, 9.75, 12.25, 3.75, 6, 0, 1);
+  }
+
+  void IsotopeDistribution::estimateFromWeightAndComp(double average_weight, double C, double H, double N, double O, double S, double P)
```

Comment:
This method is not used in `MultiplexFiltering::getAveragineSimilarity`? Where is it needed?